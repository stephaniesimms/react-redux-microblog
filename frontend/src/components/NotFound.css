.NotFound-card {
  margin: 0 auto;
}