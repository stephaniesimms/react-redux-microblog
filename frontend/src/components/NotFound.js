import React, { Component } from 'react';
import Card from 'react-bootstrap/Card';
import './NotFound.css';

/**
 * NotFou;nd: rendered when page does not match
 * any of the routes
 */

class NotFound extends Component {
  render() {
    return (
      <Card className='NotFound-card text-center'
        bg='dark'
        text='white'
        style={{ width: '30rem' }}>

        <Card.Body>
          <i className='fas fa-skull-crossbones fa-7x' />
          <Card.Title className='not-found-message pt-3'>
            Oops, we didn't find what you were looking for...
        </Card.Title>
        </Card.Body>
      </Card>
    );
  }
}
export default NotFound;
