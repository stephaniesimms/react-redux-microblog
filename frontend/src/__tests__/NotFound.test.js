import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import NotFound from '../components/NotFound';

it('renders without crashing', function () {
  shallow(<NotFound />);
});

it('matches snapshot', function () {
  let wrapper = shallow(<NotFound />);
  let serialized = toJson(wrapper);
  expect(serialized).toMatchSnapshot();
});