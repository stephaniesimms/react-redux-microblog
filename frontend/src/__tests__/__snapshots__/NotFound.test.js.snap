// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<Card
  bg="dark"
  body={false}
  className="NotFound-card text-center"
  style={
    Object {
      "width": "30rem",
    }
  }
  text="white"
>
  <CardBody>
    <i
      className="fas fa-skull-crossbones fa-7x"
    />
    <CardTitle
      className="not-found-message pt-3"
    >
      Oops, we didn't find what you were looking for...
    </CardTitle>
  </CardBody>
</Card>
`;
